docs/IconWrapper.md

```diff
@@ -53,3 +53,19 @@ const Cloud = wrapIcon({ IconClass: MaterialCommunityIcon, name: 'cloud-off-outl
 ## Notes
 
 As with all Higher Order Components, there is a performance hit if the function is called from another component's `render` method. It is therefore advised to always call `wrapIcon()` once per Icon type, and to do so outside of any methods.
+
+## Alternative to IconWrapper
+
+If you do not wish to use the HOC to wrap all of your icons individually (such as the case where you need to programmatically determine an icon to show at runtime), or you would like to use a component other than a react-native-vector-icon or PX Blue SVG, you can define your own function, so long as it matches the required shape
```

Comment:
Can we add something like "As with all HOC (Higher Order Components)..." to line 55 under notes so that when you use HOC here it is clear what you're talking about? Unless you think that would be overkill, but I wasn't sure what it was at first glance. 

Aaaaaand... can you add some punctuation to the end of this sentence? 😉